lib/people-array.js

```diff
@@ -1,21 +1,28 @@
 'use strict';
 
-// example function count function
-const count = (people) => {
-  return people.reduce((memo /*, p */) => memo + 1, 0);
-};
-
-// callback for reduce
-// each person has a gender attribute
-const isFemale = (person) => {
-
-};
+// // example function count function
+// const count = (people) => {
+//   return people.reduce((memo /*, p */) => memo + 1, 0);
+// };
+const Female = (person) => {
 
+  if(person.gender === 'f') {
+    return true;
+  } else {
+    return false;
+  }
+ };
 const women = (people) => {
-
-};
+   let womenCount = 0;
```

Comment:
Just a note: if you change the name of the variable (in this case from `women` to `womenCount`, the tests will no longer work properly.